frappe/desk/doctype/note/note.json

```diff
@@ -24,14 +25,16 @@
    "label": "Title",
    "no_copy": 1,
    "print_hide": 1,
-   "reqd": 1
+   "reqd": 1,
+   "unique": 1
```

Comment:
Can be an issue?

High chances that a public & private note title can conflict.

Reply:
This was set automatically, because title is used for naming. Should we switch to hash naming?